From dfdfd7f829f74b46f14470eae968c39b910fd11c Mon Sep 17 00:00:00 2001
From: Jon Mason <jon.mason@arm.com>
Date: Thu, 14 Dec 2023 09:01:02 -0500
Subject: [PATCH] cmake: modify path to libmetal version file

Commit ad87802d6e01e97946de20b6c2fa28aed184ed20 changed how the
versioning is done and created a version file.  Due to this change,
the VERSION file is not being found when building because the source dir
is pointing to tf-m.  Modify to point where we want it.

Upstream-Status: Inappropriate [Build workaround]

Signed-off-by: Jon Mason <jon.mason@arm.com>
---
 cmake/options.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/options.cmake b/cmake/options.cmake
index 4b866164c390..004126a40da1 100644
--- a/cmake/options.cmake
+++ b/cmake/options.cmake
@@ -1,4 +1,4 @@
-file(READ ${CMAKE_SOURCE_DIR}/VERSION ver)
+file(READ ${CMAKE_SOURCE_DIR}/../libmetal/VERSION ver)
 
 string(REGEX MATCH "VERSION_MAJOR = ([0-9]*)" _ ${ver})
 set(PROJECT_VERSION_MAJOR ${CMAKE_MATCH_1})
